andela/hc-ravens
# #156687755 Configure Nag Notifications

**What does this PR do?**
- Implement nag slider to configure nag intervals.
- Implement nag email notifications.
- Implement nag icon on the dashboard.

**Description of Task to be completed?**
Have the following endpoints working.
- A user can set the nag intervals 
- Email notifications are sent when a job is not resolved
- A nag icon appears on the dashboard when in nag mode

**How should this be manually tested?**
- After cloning the repo cd into hc-ravens folder activate virtual env.
- Log in and create a check.
- Set the Timeout, Grace and Nag sliders to 1 minute each.
- The icons for each period will change after time passes
- When in Nag mode notifications will be sent after every one minute. 

#### What are the relevant pivotal tracker stories?
#156687755
